app/src/test/java/org/simple/clinic/addidtopatient/searchforpatient/AddIdToPatientSearchScreenControllerTest.kt

```diff
@@ -0,0 +1,79 @@
+package org.simple.clinic.addidtopatient.searchforpatient
+
+import com.nhaarman.mockito_kotlin.any
+import com.nhaarman.mockito_kotlin.mock
+import com.nhaarman.mockito_kotlin.never
+import com.nhaarman.mockito_kotlin.times
+import com.nhaarman.mockito_kotlin.verify
+import io.reactivex.subjects.PublishSubject
+import junitparams.JUnitParamsRunner
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.simple.clinic.util.RxErrorsRule
+import org.simple.clinic.widgets.UiEvent
+
+@RunWith(JUnitParamsRunner::class)
+class AddIdToPatientSearchScreenControllerTest {
+
+  @get:Rule
+  val rxErrorsRule = RxErrorsRule()
+
+  private val screen: AddIdToPatientSearchScreen = mock()
+
+  private lateinit var controller: AddIdToPatientSearchScreenController
```

Comment:
We can make this a `val` and initialize it right here.
`private val controller = AddIdToPatientSearchScreenController()`

JUnit recreates the class for every test.

Reply:
Done.